### ISSUE TITLE
[ExpansionPanel] Keyboard navigation does not consider disabled state

### ISSUE THREAD

**Issue by georgi-sla** [telerik/kendo-angular#4463]
### Description

Currently, when the developer is dynamically updating the disabled state of the ExpansionPanel, the keyboard navigation fails to consider the current state and does not work as expected.

### Steps To Reproduce

1. Create an ExpansionPanel.
2. Dynamically enable/disable it.
3. Navigate to the component.
4. Try expanding/collapsing it using either Enter or Space keys.

For further reference, please check out the following StackBlitz demo: https://stackblitz.com/edit/angular-eziaqj

### Actual Behavior

When the ExpansionPanel is initially enabled and the developer programmatically disables it, the keyboard navigation remains active and allows the user to expand/collapse the component even in cases when it is disabled.
On the other hand, when the component is initially disabled and the developer enables it, the keyboard navigation should also be active and allow the user to expand/collapse the element.

### Expected Behavior

The keyboard navigation should correctly consider the state of the ExpansionPanel and should work accordingly - disabled in a disabled component, and enabled in an enabled component.

### Browser

All

### Browser version

latest

### OS type

MacOS

**Comment by Raisolution** [comment]
Fix available in `v17.1.1`
